src/lib/config/protocol.ts

```diff
@@ -0,0 +1,55 @@
+export const protocolConfig = {
+  peg: {
+    asset: "Gold",
+    adjective: "Gold-Pegged",
+    description: "digital gold-pegged tokens",
+  },
+  tokens: {
+    /**
+     * Neutron = stable token in the Gluon protocol.
+     * For the current deployment this corresponds to GAU.
+     */
+    neutron: {
+      ticker: "GAU",
+      name: "Gluon Gold",
+      shortDescription: "Stable token pegged to 1g of Gold.",
+    },
+    /**
+     * Proton = volatile token in the Gluon protocol.
+     * For the current deployment this corresponds to GAUC.
+     */
+    proton: {
+      ticker: "GAUC",
+      name: "Gluon Gold Certificate",
+      shortDescription: "Volatile token that tokenizes the reserve surplus.",
+    },
+  },
+  seo: {
+    defaultTitle: "Gluon | Gold Protocol on Ergo",
+    defaultDescription:
+      "Gluon is a decentralized finance protocol on Ergo blockchain that enables users to trade, swap, and manage gold-backed tokens.",
+    defaultKeywords: "Gluon, Ergo, DeFi, Blockchain, Cryptocurrency, Gold-backed tokens, GAU, GAUC, Digital Assets",
+    defaultUrl: "https://www.gluon.gold/",
+    defaultImage: "/logo/gluon.png",
+  },
+  ui: {
+    hero: {
+      tagline: "Trade and transact with digital gold-pegged tokens.",
+      subTagline: "Secured by the Ergo blockchain and its decentralized gold price oracle.",
+    },
+    labels: {
+      navbarTitle: "GLUON GOLD",
+      telegramHandle: "GluonGold",
+      assetPriceTitle: "1 kg of Gold",
```

Comment:
This line and line 44 seem to be using a text from an old version. See https://gluon.gold to see the text that we are using now.